File: OlprrApi/OlprrApi.Models/Response/Pcs.cs
using System;

namespace OlprrApi.Models.Response
{
    public class Pcs
    {
        public int PcsId { get; set; }
        public int? DispLandfillFacId { get; set; }
        public int? DispThrmFacId { get; set; }
        public Boolean? OnSiteDisposalInd { get; set; }
        public string OffSiteDispAddr { get; set; }
        public int? SoilStatusId { get; set; }
        public string SoilStatusCode { get; set; }
        public decimal? LandfillAmtYrds3 { get; set; }
        public decimal? ThermalAmtYrds3 { get; set; }
        public decimal? OnSiteAmtYrds3 { get; set; }
        public decimal? OffSiteAmtYrds3 { get; set; }
        public DateTime? DisposalDate { get; set; }
        public string PcsComment { get; set; }
        public string LastChangeBy { get; set; }
        public DateTime? LastChangeDate { get; set; }
        public int LustId { get; set; }
        public string LogNumber { get; set; }
        public string SwlaPermitNbr { get; set; }
    }
}

File: osearch/src/app/models/pcs.ts
export class Pcs {
    pcsId: number;
    dispLandfillFacId?: number;
    dispThrmFacId?: number;
    onSiteDisposalInd?: boolean;
    offSiteDispAddr: string;
    soilStatusId?: number;
    soilStatusCode?: string;
    landfillAmtYrds3?: number;
    thermalAmtYrds3?: number;
    onSiteAmtYrds3?: number;
    offSiteAmtYrds3?: number;
    disposalDate: Date;
    pcsComment: string;
    lastChangeby: string;
    lastChangeDate: Date;
    lustId: number;
    logNumber: string;
    swlaPermitNbr: string;
}
